src/PlotRecipes/ElectricField.jl

```diff
@@ -16,14 +17,15 @@
     unitformat --> :slash
 
     ef_magn = norm.(ef)
-    ElectricPotential(ef_magn, grid), cross_section, idx, value, internal_efield_unit, contours_equal_potential, full_det
+    ElectricPotential(ef_magn, grid), cross_section, idx, value, internal_efield_unit, contours_equal_potential, full_det, all_point_types_bits
```

Comment:
```suggestion
    ElectricPotential(ef_magn, grid), cross_section, idx, value, internal_efield_unit, contours_equal_potential, full_det, false
```
But in general, I would be strongly in favor in reverting this and to deal with this in the `PointTypes` plot-specific function